operator/cmd/main.go

```
@@ -0,0 +1,213 @@
+		host := os.Getenv(prefix + "HOST")
+		if host == "" {
+			continue // engine not configured; that is fine
+		}
+
+		adminSecret := os.Getenv(prefix + "ADMIN_SECRET")
+		ns, name, ok := strings.Cut(adminSecret, "/")
+		if !ok || ns == "" || name == "" {
+			return nil, fmt.Errorf("%sADMIN_SECRET must be namespace/name, got %q", prefix, adminSecret)
+		}
+
+		port := int32(defaultPort(e))
```

golangci-lint (2.12.2)

[high] G115: integer overflow conversion int -> int32

(gosec)